javascript/javascript3/week1/Giphy/giphy.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <style>
+      body {
+        margin-top: 10%;
```

Comment:
Would be great if you move the styles in the separated file.

Reply:
yes offcourse, wasnt much so thought to just populate it right there.